src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Application {
+    private static final int ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[ARRAY_SIZE];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i <= ARRAY_SIZE / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else if (i > ARRAY_SIZE / 2) {
```

Comment:
You don't need to check `i > ARRAY_SIZE / 2` in the `else if` statement. Since the first `if` condition checks if `i` is less than or equal to `ARRAY_SIZE / 2`, the `else` part will only run when `i` is greater than `ARRAY_SIZE / 2`. Therefore, you can simply use `else` instead of `else if (i > ARRAY_SIZE / 2)`.